src/main/java/rs/teslaris/core/model/commontypes/GeoLocation.java

```diff
@@ -22,4 +23,22 @@ public class GeoLocation {
 
     @Column(name = "precision_in_meters", nullable = false)
     private int precisionInMeters;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
```

Comment:
I think that this equals implementation can be generated with @Equals lombok snnotation at class level.